components/sidebar/sidebar_channel/sidebar_channel_close/index.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+import {bindActionCreators, Dispatch, ActionCreatorsMapObject} from 'redux';
+
+import {Channel} from 'mattermost-redux/types/channels';
+import {GlobalState} from 'mattermost-redux/types/store';
+import {ActionFunc} from 'mattermost-redux/types/actions';
+
+import SidebarChannelClose from './sidebar_channel_close';
+
+type OwnProps = {
+    channel: Channel;
+    show: boolean;
+}
+
+function mapStateToProps(state: GlobalState, ownProps: OwnProps) {
```

Comment:
Can you remove the reference to `ownProps`? Just having it defined actually causes `react-redux` to memoize differently

Reply:
Do we even need to have this? Can't we pumm null to connect?

Reply:
Yeah i'm going to remove the connector here as it isn't needed right now.